src/Benchmarks/Program.cs

```diff
@@ -125,10 +125,18 @@ public static void Main(string[] args)
             }
             else if (String.Equals(Server, "IISInProcess", StringComparison.OrdinalIgnoreCase))
             {
+                // if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ASPNETCORE_IIS_VERSION")))
```

Comment:
This relies on an 8.0 feature which isn't available in the old pinned version we currently have (see previous comment). We can uncomment once we've unpinned the version.